snuba/settings_docker.py

```diff
@@ -13,5 +13,7 @@
 USE_REDIS_CLUSTER = False
 
 # Dogstatsd Options
-DOGSTATSD_HOST = None
-DOGSTATSD_PORT = None
+DOGSTATSD_HOST = os.getenv("DOGSTATSD_HOST")
+DOGSTATSD_PORT = os.getenv("DOGSTATSD_PORT")
+
+SENTRY_DSN = os.getenv("SENTRY_DSN")
```

Comment:
Any reason for not using the os.environ.get as in the lines above? Keeping these consistent would help maintainability as nobody would need to ask why some settings are loaded with os.getenv and others are not.

Reply:
No actual reason and changed to be consistent with other ones. TnX